### DISCUSSION TITLE
欠缺人力巡檢軌道？AI能幫忙

### DISCUSSION THREAD

**Discussion by lukiea** [1101-GS3073-C/Python-Community-GS3073-C#711] [General]
https://futurecity.cw.com.tw/article/349
人力漸漸被淘汰，ＡＩ　將變成人類最好的工具

**Comment by yjpxi** [comment]
AI 系統可以自動化地檢測和分析軌道的狀況，無需人工介入進行日常檢查，雖然減少了很多鐵道事故發生，但是自動化系統會取代部分人工巡檢的工作，造成部分工作人員的失業。
